apps/dashboard/lib/get-all-transactions.ts

```diff
@@ -0,0 +1,37 @@
+import { TransactionEdge } from "@/services/graphql/generated"
+import {
+  fetchFirstTransactions,
+  fetchPaginatedTransactions,
+} from "@/services/graphql/queries/get-transactions"
+
+export async function fetchAllTransactionsByCount({
```

Comment:
```suggestion
export async function fetchTransactionsByCount({
```

naming is misleading. you won't fetch all if you are asking for a count number